File: fixtures/scale/java/medium/src/calc/ast/VariableExpr.java
package calc.ast;

public class VariableExpr implements Expr {
    private final String name;

    public VariableExpr(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public String describe() {
        return name;
    }
}

File: fixtures/scale/dart/medium/lib/ast/variable_expr.dart
import 'expr.dart';

class VariableExpr implements Expr {
  final String name;

  VariableExpr(this.name);

  @override
  String describe() => name;

  bool get isConstantName => const {'pi', 'e', 'phi', 'tau'}.contains(name);
}
